dockstore-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -1625,6 +1629,58 @@ private Workflow registerGatkSvWorkflow(WorkflowsApi ownerWorkflowApi) {
                 "test", "wdl", "/test.json");
         return ownerWorkflowApi.refresh(workflow.getId());
     }
+    
+    @Test
+    public void testWorkflowVersionAliasOperations() throws ApiException {
+        final ApiClient webClient = getWebClient(USER_2_USERNAME, testingPostgres);
+        WorkflowsApi workflowApi = new WorkflowsApi(webClient);
+        workflowApi.manualRegister("github", "DockstoreTestUser2/dockstore_workflow_cnv",
+                "/workflow/cnv.cwl", "", "cwl", "/test.json");
+        final Workflow workflowByPathGithub = workflowApi.getWorkflowByPath(DOCKSTORE_TEST_USER2_RELATIVE_IMPORTS_WORKFLOW, null, false);
+        // do targetted refresh, should promote workflow to fully-fleshed out workflow
+        final Workflow workflow = workflowApi.refresh(workflowByPathGithub.getId());
+        workflowApi.publish(workflow.getId(), SwaggerUtility.createPublishRequest(true));
+
+        Assert.assertTrue(workflow.getWorkflowVersions().stream().anyMatch(versions -> "master".equals(versions.getName())));
+        Optional<WorkflowVersion> optionalWorkflowVersion = workflow.getWorkflowVersions().stream()
+                .filter(version -> "master".equalsIgnoreCase(version.getName())).findFirst();
+        assertTrue(optionalWorkflowVersion.isPresent());
+        WorkflowVersion workflowVersion = optionalWorkflowVersion.get();
+
+        // give the workflow version a few aliases
+        AliasesApi aliasesApi = new AliasesApi(webClient);
+        WorkflowVersion workflowVersionWithAliases = aliasesApi.addAliases(workflowVersion.getId(), "awesome workflowversion, spam, test workflowversion");
+        Assert.assertTrue("entry is missing expected aliases",
+                workflowVersionWithAliases.getAliases().containsKey("awesome workflowversion")
+                        && workflowVersionWithAliases.getAliases().containsKey("spam")
+                        && workflowVersionWithAliases.getAliases().containsKey("test workflowversion"));
+
+        Assert.assertNotNull("Getting workflow by ID has null alias", workflowVersionWithAliases.getAliases());
```

Comment:
This is redundant; if getAliases() returned null, you would have already gotten an NPE on line 1654.

Reply:
Removed extraneous assert